frontend/packages/kubevirt-plugin/src/constants/vm/constants.ts

```diff
@@ -16,9 +16,11 @@ export const TEMPLATE_PARAM_VM_NAME_DESC = 'Name for the new VM';
 export const TEMPLATE_TYPE_LABEL = 'template.kubevirt.io/type';
 export const TEMPLATE_TYPE_VM = 'vm';
 export const TEMPLATE_TYPE_BASE = 'base';
+export const TEMPLATE_TYPE_APP = 'app';
```

Comment:
please rename to `APP` or we can use a simple string over there - but it is not a template type